test/built-ins/Array/from/mapfn-throws-exception.js

```diff
@@ -0,0 +1,25 @@
+// Copyright 2015 Microsoft Corporation. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*---
+description: mapFn throws an exception
+es6id: 22.1.2.1
+---*/
+
+var array = [ 2, 4, 8, 16, 32, 64, 128 ];
+function mapFn(value, index, obj) {
+    throw "Expected error";

```

Comment:
If you use `throw new Test262Error();` here, then you can express this test's assertion more simply:

```js
assert.throws(Test262Error, function() {
  Array.from(array, mapFn);
});
```

Note that there is no need to test the return value in this case, since if the CallExpression throws an exception, we can take it for granted that no assignment would follow.